upload/catalog/model/extension/shipping/zasilkovna.php

```diff
@@ -180,6 +199,17 @@ private function calculatePrice($countryCode, $totalWeight, $totalPrice) {
 		];
 	}
 
+    /**
+     * Gets cart weight in kilograms.
+     *
+     * @return double
+     */
+    private function getCartWeightKg()
+    {
+        $weightClassRow = $this->localisationWeightClassModel->getWeightClassDescriptionByUnit('kg');
+        return (double) $this->weight->convert($this->cart->getWeight(), $this->config->get('config_weight_class_id'), $weightClassRow['weight_class_id']);
```

Comment:
Proč double?

Reply:
Protože to bylo použité v původním kódu při ukládání objednávky. A protože je double typ uvedený v PHPDocích metod aplikující pravidla přepravních metod. Takže double je více konsistentní s okolním kódem. Mezi double, float a real není v PHP rozdíl.

Reply:
Přesně proto se ptám. Použijme float, je to best practice.

Reply:
Odkuď máš to, že float je best practice? Dej odkaz. Vím, že lidi místo float používají string a bcmath funkce.